flambe/sampler/base.py

```diff
@@ -264,15 +274,24 @@ def sample(self,
         if len(data) == 0:
             raise ValueError("No examples provided")
 
-        if self.downsample:
-            if not (0 < self.downsample <= 1):
+        if self.downsample or self.downsample_max_samples:
```

Comment:
we may want to use is not None here in case people use 0, even if that s weird

Reply:
done